dart/test/test_utils.dart

```diff
@@ -303,12 +303,12 @@ void runTest({Codec<List<int>, List<int>> gzip, bool isWeb = false}) {
     try {
       throw ArgumentError('Test error');
     } catch (error, stackTrace) {
-      final response = await client.captureException(
-          exception: error, stackTrace: stackTrace);
-      expect(response.isSuccessful, false);
-      expect(response.eventId, null);
-      expect(
-          response.error, 'Sentry.io responded with HTTP 401: Invalid api key');
+      final response =
```

Comment:
we could rename the `response` to sentryId as well

Reply:
✅